### DISCUSSION TITLE
Is it possible to use external relay for outgoing messages delivery?

### DISCUSSION THREAD

**Discussion by rooty123** [emersion/maddy#585] [General]
Hi
For example I want to use AWS SES to handle all outgoing messages. In fact I do it just from email client, but it is not convenient to set up clients to use different creds for IMAP and SMTP. 

My goal is use single creds for SMTP and IMAP but still want to relay all outgoing messages to AWS SES, since it takes care about DKIM and other things that I don't want to care about.

**Comment by nickersonm** [comment]
I don't know about AWS SES specifically, but you can use an outgoing relay with the [`target.smtp` directive](https://maddy.email/reference/targets/smtp/#configuration-directives) as long as it supports PLAIN or TLS certificate authentication. For example, I've used [Sendgrid](https://docs.sendgrid.com/for-developers/sending-email/ssmtp) successfully:
```conf
# Use SendGrid SMTP relay for outbound mail
# Replaces target.remote section
target.smtp outbound_delivery {
  # Only allow TLS connection
  attempt_starttls yes
  require_tls yes
  # Authentication method
  # Sendgrid has user = "apikey" and password = "<key">
  auth plain apikey <key>
  # SMTP server
  targets tcp://smtp.sendgrid.net:587
}
```

**Reply by rooty123** [reply]
Thanks, that is what I was looking for
